Desktop/components/JASP/Widgets/DataTableView.qml

```diff
@@ -222,7 +222,40 @@ FocusScope
 						event.accepted = true;
 						mainWindowRoot.changeFocusToFileMenu();
 						break;
-
+									
+				
+				case Qt.Key_PageUp:
+					event.accepted = true;
+					if(shiftPressed)	contentX = Math.max(0, contentX - width)
+					else				contentY = Math.max(0, contentY - height)
+					break;
+								
+				case Qt.Key_PageDown:
+					event.accepted = true; 
+					if(shiftPressed)	contentX = Math.min(contentWidth  - width,  contentX + width)
+					else				contentY = Math.min(contentHeight - height, contentY + height)
+					break;
+				
+				case Qt.Key_Down:
```

Comment:
these dont seem to work, neither do the ones in the JASPDataView..

But this isnt a regression and I dont really want to spend more time on this issue.
In fact, the feature request is now implemented so that is something.